.hermes/conveyor/work-028d4710/adr.md

```
@@ -0,0 +1,63 @@
+# ADR: Native Rust `PerlLanguage` Descriptor for `tree-sitter-perl-rs`
+
+## Status
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)